Cargo.toml

```diff
@@ -24,3 +24,5 @@ anyhow = { version = "1.0" }
 google-signin = { git = "https://github.com/hi-rustin/google-signin-rs" }
 rand = "0.8.4"
 either = "1.6"
+lazy_static = "1.4.0"
```

Comment:
Why not just use oncecell's lazy?

Reply:
Just copy the example of [regex](https://github.com/rust-lang/regex#usage-avoid-compiling-the-same-regex-in-a-loop) 🤣, seems oncecell's lazy is also ok here. I have updated it.